src_cam/physpkg.F90

```diff
@@ -2728,9 +2840,38 @@ subroutine tphysbc (ztodt, state,  &
 
     call t_stopf('bc_init')
 
+    if (compute_enthalpy_flux) then
+       call cnst_get_ind('Q', ixq)
+       call cnst_get_ind('CLDLIQ', ixcldliq)
+       call cnst_get_ind('CLDICE', ixcldice)
```

Comment:
Bit of potential cleanup:
Move `ixq`, `ixcldliq` and `ixcldice` to be module variables, move the `cnst_get_ind` calls to `phys_init` (including from `tphysbc`), and remove them from the run-time routines.